generate.js

```diff
@@ -18,9 +18,9 @@ const renderPage = (src, dst, resources) => {
 // events
 const events_url = "https://ccns.kktix.cc/events.json";
 const renderEvents = (src, dst) => {
-  request.get(events_url, (error, response, body) => {
-    if (!error && response.statusCode == 200) {
-      var events = JSON.parse(body);
+  axios.get(events_url)
+  .then((response) => {
+      var events = response.data;
```

Comment:
It might be better to just unindent it. `git diff` has an option to ignore indentation changes.